b2share/config.py

```diff
@@ -458,3 +639,13 @@
 #: There is no password so don't send password change emails
 SECURITY_SEND_PASSWORD_CHANGE_EMAIL=False
 SECURITY_SEND_PASSWORD_RESET_NOTICE_EMAIL=False
+
+# Extra (Harry Kodden)
+# When testing in HTTP, both cookie secure and CSRF enforcement is switched off
+APP_ENABLE_SECURE_HEADERS = (os.environ.get('SESSION_COOKIE_SECURE', "True").upper() == "True".upper())
+
+SESSION_COOKIE_PATH="/"
+SESSION_COOKIE_SAMESITE="Lax"
```

Comment:
`SESSION_COOKIE_SAMESITE` comes from Flask: https://flask.palletsprojects.com/en/1.1.x/config/#SESSION_COOKIE_SAMESITE

For more info on possible values see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite